### ISSUE TITLE
Unable to compile my project with ember-click-outside version 1.2.0

### ISSUE THREAD

**Issue by ghost** [zeppelin/ember-click-outside#54]
Hi, I have been using `ember-click-outside` in my project with version `1.0.2`, and now I'm trying to update to `1.2.0` but the project "compilation" fails with the following error:

![image](https://user-images.githubusercontent.com/47750606/67299035-84c75700-f4c2-11e9-8b09-fe69094db13f.png)

I'm using the following ember versions:
```log
DEBUG: ------------------------------- index.js:185
DEBUG: Ember             : 3.12.0 index.js:185
DEBUG: Ember Data        : 3.12.0 index.js:185
DEBUG: jQuery            : 3.4.1 index.js:185
DEBUG: Ember Simple Auth : 1.9.1 index.js:185
DEBUG: -------------------------------
```
Thanks in advance!

**Comment by zeppelin** [comment]
@go-estebanborai that's quite strange. I'll look into it, although it doesn't seem like it's an issue with ember-click-outside...

**Comment by go-estebanborai** [comment]
@zeppelin Thanks!

**Comment by estebanborai** [comment]
Hi @zeppelin !

I'm writting from my personal account, I used to have `go-estebanborai` account.
I wanted to close this issue as the problem is solved now.

Looks like `ember-oo-modifier` had conflicts with this package for some reason. After removing `ember-oo-modifier` and installing `ember-modifier`, I was able to install the latest version of `ember-click-outside` (v1.3.0).

Thanks for your time!
